app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+from .cafe import Cafe
+from .errors import NotWearingMaskError, VaccineError
```

Comment:
Use absolute imports instead of relative imports as per the checklist. Change to: `from app.cafe import Cafe` and `from app.errors import NotWearingMaskError, VaccineError`.